app/app.go

```diff
@@ -191,9 +191,11 @@ type App struct {
 	BasicManager  module.BasicManager
 	ModuleManager *module.Manager
 	configurator  module.Configurator
-	// blockTime is used to override the default TimeoutHeightDelay. This is
-	// useful for testing purposes and should not be used on public networks
-	// (Arabica, Mocha, or Mainnet Beta).
+
+	// txValidationCache caches expensive blob transaction validation results
```

Comment:
[nit]
expensive might be misleading here with the fees
```suggestion
	// txValidationCache caches blob transaction validation results
```